lib/network/level-net/utils.js

```diff
@@ -0,0 +1,30 @@
+'use strict'; // eslint-disable-line
+
+module.exports.serializeError = function serializeError(err) {
+    if (!err) {
+        return err;
+    }
+    const serializedErr = Object.assign({}, err);
+
+    serializedErr.message = err.message;
+    for (const k in err) {
+        if (!(k in serializedErr)) {
+            serializedErr[k] = err[k];
+        }
+    }
```

Comment:
Some properties are not copied by Object.assign (don't ask me why exactly, probably related to computed properties). Anyway, the loop is able to catch those properties, e.g. not doing it fails a few tests like in CRUD test when checking `err.notFound`.